rules/do-you-know-how-to-use-connection-strings/rule.md

```diff
@@ -10,172 +10,38 @@ authors:
   url: https://ssw.com.au/people/adam-cogan
 - title: William Liebenberg
   url: https://ssw.com.au/people/william-liebenberg
+- title: Bryden Oliver
+  url: https://ssw.com.au/people/bryden-oliver
 - title: Calum Simpson
   url: https://ssw.com.au/people/calum-simpson
 - title: Ryan Tee
   url: https://ssw.com.au/people/ryan-tee
 related: []
 redirects: 
 - do-you-know-how-to-use-connection-string-in-net-2-0
+- do-you-hard-code-your-connectionstring
+- know-the-right-way-to-define-a-connection-string
 
 ---
 
 Accessing your application configuration and secret values is easily done via the `IConfiguration` interface (from `Microsoft.Extensions.Configuration.Abstractions`).
 
 However, this convenience can lead you down the path of loosely typed secret handling (everything is a `string`) and can cause maintenance overhead. This is bad!
```

Comment:
These first 2 paragraphs feel out of place - I think it reads better without them.

Reply:
maybe have some bad examples